scout/server/blueprints/variant/utils.py

```diff
@@ -181,55 +167,62 @@ def update_variant_case_panels(store, case_obj, variant_obj):
     variant_obj["case_panels"] = case_panel_objs
 
 
-def add_gene_info(store, variant_obj, gene_panels=None, genome_build=None):
+def get_extra_info(gene_panels: List) -> Dict:
```

Comment:
```suggestion
def get_extra_info(gene_panels: list) -> Dict[int, dict]:
```

Reply:
I have a feeling I will regret this, as there are some old mentions of hgnc_id as a `str` still, but lets go - it is the way we do it now.

Reply:
This scenario shouldn't be a problem. Type hinting is just for documentation, so even if the function produces Dict[str, dict] it won't trigger an error, because the output is not validated against the expected hints

Reply:
https://stackoverflow.com/questions/67412233/python-type-hinting-not-generating-error-for-wrong-type